grpc-client-server/prober/prober.proto

```diff
@@ -12,10 +12,13 @@ service Prober {
 // The request message 
 message ProbeRequest {
   string endpoint = 1;
+  int32 numberOfRequests = 2;
 }
 
 // The response message containing the result
+// returns the average response time
 message ProbeReply {
-  float result = 1;
+  float averageResponseTime = 1;
```

Comment:
For times, if we store them as primitive types (rather than a language-specific duration type), we tend to include the units in the name of the field to make clear what 1 means. Is it 1 second? 1 millisecond? 1 hour? 1 day?